profiles/views_test.py

```diff
@@ -14,14 +14,24 @@
 )
 
 from backends.edxorg import EdxOrgOAuth2
+from courses.factories import ProgramFactory
+from dashboard.models import ProgramEnrollment
 from profiles.factories import ProfileFactory, UserFactory
 from profiles.models import Profile
-from profiles.permissions import CanEditIfOwner
+from profiles.permissions import (
+    CanEditIfOwner,
+    CanSeeIfNotPrivate,
+)
 from profiles.serializers import (
     ProfileLimitedSerializer,
     ProfileSerializer,
 )
 from profiles.views import ProfileViewSet
+from roles.models import (
+    Instructor,
+    Role,
+    Staff,
+)
```

Comment:
please import `Instructor` and `Staff` from the proper module
`from roles.roles import Staff, Instructor`